libcxx/test/libcxx/system_reserved_names.gen.py

```diff
@@ -158,5 +158,8 @@
 #define erase SYSTEM_RESERVED_NAME
 #define refresh SYSTEM_RESERVED_NAME
 
+// Macros from the CUDA SDKs
+#define __noinline__ __attribute__((noinline))
```

Comment:
I believe we want to make this macro be defined as SYSTEM_RESERVED_NAME to prove we're not actually using it.

Reply:
Indeed, fixed.